bionic/dagviz.py

```diff
@@ -14,6 +14,26 @@
 Image = import_optional_dependency('PIL.Image', purpose=module_purpose)
 
 
+class SvgImage(object):
+    def __init__(self, xml_string):
+        self.xml_string = xml_string
+
+    def save(self, fp):
+        '''
+        Save SVG to a file
```

Comment:
We should probably document what types are supported for `fp`. It looks like we support filenames and Paths, but not file objects (unlike `Image.save`)?

Also, a nit: the convention in the project is to still to complete sentences with periods in docstrings.